src/System.Private.ServiceModel/src/System/ServiceModel/Channels/RTHttpClientHandler.cs

```diff
@@ -0,0 +1,475 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
I believe the new naming convention is <file><group>.cs, so I believe this should probably be HttpClientHandler.netcore50.cs.  Eventually we need to rename our other RTxxx.  I see not all CoreFx projects are consistent -- some use file.NETNative.cs.